lib/PlaidClient.js

```diff
@@ -479,31 +479,21 @@ Client.prototype.listPaymentRecipients =
 
 // createPayment(String, String, Object, Function)
 Client.prototype.createPayment =
-  function(recipient_id, reference, amount, cb) {
+  function(recipient_id, reference, amount, options, cb) {
+    if (typeof options === 'function') {
```

Comment:
Given that _authenticatedRequest already has this check (see line 76), can we just pass the args directly into the function as-is?